kevm-pyk/src/tests/unit/test_unit.py

```diff
@@ -206,7 +206,8 @@ def test_method_sig_from_abi() -> None:
     "aggregate((address,bytes)[])": "252dba42"
   },
   "ast": {
-    "absolutePath": "lib/forge-std/src/interfaces/IMulticall3.sol"
+    "absolutePath": "lib/forge-std/src/interfaces/IMulticall3.sol",
+    "nodes": []
   },
```

Comment:
Do we need to adjust `tests/foundry/foundry.toml` to include this extra output?

Reply:
I think what's happening here is that we're already specifying the `"ast"` to be included, and the `"nodes"` is a subobject of that which should be generated whenever `"ast"` is. For the `foundry.toml` file, I haven't been able to find any documentation confirming this, but it seems the `"ast"` is generated by default when calling the solidity compiler through forge. The reason I had to change this test was that it was manually constructed and didn't have this json field that our constructors now expect.